modules/sagemaker/sagemaker-templates-service-catalog/common/code_repo_construct.py

```diff
@@ -0,0 +1,293 @@
+from enum import Enum
+from typing import Any
+
+from aws_cdk import Aws, CustomResource, Duration
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_lambda as lambdafunction
+from constructs import Construct
+
+
+class RepositoryType(str, Enum):
+    CODECOMMIT = "CodeCommit"
+    GITHUB = "GitHub"
+    GITLAB = "GitLab"
+    GITHUB_ENTERPRISE = "GitHub Enterprise"
+    GITLAB_ENTERPRISE = "GitLab Enterprise"
```

Comment:
Can you confirm GitLab is supported at this time? There is no mention of that in the readme

Reply:
For now, removed GitLab from supported repository types until we enable integration for it